src/resourcetiming-decompression.js

```diff
@@ -133,7 +143,9 @@
         }
 
         var initiatorType = parseInt(data[0], 10);
-        var timings = data.length > 1 ? data.substring(1).split(",") : [];
+        data = data.length > 1 ? data.split(SPECIAL_DATA_PREFIX + SPECIAL_DATA_SIZE_TYPE) : [];
```

Comment:
Hm I don't think this will work.  There will be 0 or 1 dimensions special value, and 0 or 1 sizes special value.  If there's a sizes special value, it should apply to all other timings for this URL.